pkg/fileutils/read.go

```diff
@@ -38,17 +39,22 @@ func Read(file string, base string) ([]byte, error) {
 }
 
 func readAbsFile(file string) ([]byte, error) {
+	// The file argument could be an abs file reference or a URL
 	u, err := url.Parse(file)
 	if err != nil {
 		return nil, err
 	}
-	if u.Scheme == "" {
+	if u.Scheme == "http" || u.Scheme == "https" || u.Scheme == "file" {
+		return ReadUrl(u, 0)
+	}
+	// On Windows the URL potentially ends up with the drive letter as the scheme so compare with the URL as a string
+	if u.Scheme == "" || (runtime.GOOS == "windows" && len(u.Scheme) == 1 && strings.HasPrefix(u.String(), u.Scheme + ":")) {
```

Comment:
Nit: maybe extract the Windows-specific check in a small aptly-named function?

I was initially tempted to check that `u.Scheme` belongs to [a-z], but today I learnt that Windows XP (and later versions) supports mounting non-letter drives ([source](https://en.wikipedia.org/wiki/Drive_letter_assignment#Common_assignments)).